Libraries/LibWeb/HTML/HTMLTrackElement.cpp

```diff
@@ -232,6 +236,9 @@ void HTMLTrackElement::start_the_track_processing_model_parallel_steps(JS::Realm
         m_fetch_algorithms = Fetch::Infrastructure::FetchAlgorithms::create(vm(), move(fetch_algorithms_input));
         m_fetch_controller = MUST(Fetch::Fetching::fetch(realm, request, *m_fetch_algorithms));
         return;
+    } else {
+        fire_error_event();
```

Comment:
Ah, pretty weird that the spec hid the "or if URL is the empty string" in some prose, rather than as an actual step after step 10.